gen-rpm.in

```diff
@@ -165,6 +165,7 @@ if [ "$DOCKERSTATUS" == "FALSE" ]; then
 	echo "Moving completed RPM package into CMake build folder."
 elif [ "$DOCKERSTATUS" == "TRUE" ]; then
 	# Move into pickup location
+	mkdir ~/root/sunshine-build/package-rpm/sunshine.rpm
```

Comment:
I think we want this. Right?

`mkdir -p ~/root/sunshine-build/package-rpm`

Reply:
No. `~` is `/root` because root is running it. `~/root/sunshine-build/package-rpm` assumes `/root/root/sunshine-build/package-rpm`.

Reply:
My point was to add the `-p` argument, and don't include the file name, only the directory name.

Reply:
Yeah. Also didn't mean to put in the tilde. Whoops.

Reply:
> My point was to add the `-p` argument, and don't include the file name, only the directory name.

I read your comment too fast. Sorry.

Reply:
And remove this `/sunshine.rpm` from the end

Reply:
Done. Had a long day today; you can tell I'm running on fumes.

Reply:
> Done. Had a long day today; you can tell I'm running on fumes.

All good, same for me... Hoping to get this one over the goal line